metrics-dropwizard/src/main/resources/reference.conf

```diff
@@ -0,0 +1,63 @@
+usi.metrics {
+
+  # A prefix that is used when constructing metric names
+  name-prefix: "usi"
+
+  # Histogram realated settings
+  histogram {
+    # The highest trackable value of histograms and timers
+    reservoir-highest-trackable-value: 3600000000000
+
+    # The number of significant decimal digits to which histograms and timers will maintain value resolution and separation
+    reservoir-significant-digits: 3
+
+    # Clear histograms and timers fully according to the given interval
+    reservoir-reset-periodically: true
+
+    # A histogram resetting interval
+    reservoir-resetting-interval: 5000ms
+
+    # Histogram reservoirs are divided into this number of chunks, and one chunk is cleared after each (resetting interval / number of chunks) elapsed
+    reservoir-resetting-chunks: 0
+  }
+
+  # Enable StatsD reporter. If enabled, the `statsd-reporter` block should be enabled and
+  # host and port should be provided
+  statsd-reporter: off
+
+  # StatsD reporter
+  # reporters.statsd {
+  #   # A remote hostname for the StatsD reporter
+  #   host:
+  #
+  #   # A remote port for the StatsD reporter
+  #   port:
+  #
+  #   # A transmission interval for the StatsD reporter.
+  #   transmission-interval: 10000ms
+  # }
+
+  # Enable DataDog reporter. If enabled, the `datadog-reporter-on` block should be enabled
+  datadog-reporter: off
+
+  # DataDog reporter
+  #  reporters.datadog {
+  #    # A remote hostname for the DataDog reporter.
+  #    host:
+  #
+  #    # A remote port for the DataDog reporter.
+  #    port: 0
+  #
+  #    # A protocol to use with the DataDog reporter (default: udp; supported protocols: udp, api).
+  #    # When using "api" protocol, `api-key` should be provided. With "udp" protocol  host and port
+  #    # should be set.
+  #
+  #    protocol: "api"
+  #
+  #    # A DataDog API key
+  #    api-key: "FAKE-API-KEY"
```

Comment:
This should be an environment variable. We should not encourage users to set a key in any text file unless it's encrypted such as credentials in Travis CI files.

Reply:
Good catch. It was and then I replaced it with this fake string for tests.